src/omen/cogs/wvw_commands.py

```diff
@@ -7,6 +7,17 @@
 from discord.ext import commands
 from discord import app_commands
 from functions import to_lower, attach_image
+import numpy as np
+import pandas
+from pandas import DataFrame
+from pandas import RangeIndex
+from sklearn.neighbors import KNeighborsRegressor
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.ensemble import GradientBoostingRegressor
+from sklearn.tree import DecisionTreeRegressor
```

Comment:
I'm going to clean these up once I settle on an exact regressor. Need to do some more testing and tuning, but I'd like to get it pushed out and see it working in the mainline just to make sure this is useful while I'm doing that testing and tuning.